db/seeds.ts

```diff
@@ -51,6 +51,7 @@ const seed = async () => {
     { wikidata: "Q59094171", name: "Ethical review", schema: "CreativeWork" },
     { wikidata: "Q580922", name: "Preprint", schema: "CreativeWork" },
     { wikidata: "Q1744627", name: "Classification algorithm", schema: "CreativeWork" },
+    { wikidata: "Q642946", ™name: "Gamebook", schema: "CreativeWork" },
   ]
```

Comment:
```suggestion
    { wikidata: "Q642946", name: "Gamebook", schema: "CreativeWork" },
```

Reply:
That is a good catch! Very odd that that got in there 😅